terms.html

```diff
@@ -62,17 +73,17 @@
   <dt><dfn data-lt="holders|holder's|holders'">holder</dfn></dt>
   <dd>
 A role an <a>entity</a> might perform by possessing one or more
-<a>verifiable credentials</a> and generating <a>presentations</a> from them.
-A holder is usually, but not always, a <a>subject</a> of the <a>verifiable
-credentials</a> they are holding. Holders store their <a>credentials</a> in
-<a>credential repositories</a>.
+<a>verifiable credentials</a> and generating <a>verifiable presentations</a> from them.
```

Comment:
```suggestion
<a>verifiable credentials</a> and creating <a>verifiable presentations</a> from them.
```

The language that won the "naming poll" was "create"... so, we're trying to align language such that we say things like "a holder creates a verifiable presentation from one or more verifiable credentials".